### PR TITLE
Hub container

### PR DESCRIPTION
**Pull request by cbudz**
Adding files for the Hub container workflow. 

Preview here: http://file.rdu.redhat.com/cbudzilo/containers/build/tmp/en-US/html-single/

### CODE REVIEW

File: downstream/assemblies/hub/assembly-populate-container-registry.adoc [hunk 1]
```diff
@@ -10,13 +10,14 @@ ifdef::context[:parent-context: {context}]
 
 
 [role="_abstract"]
-By default, private {HubName} does not include container images. In order to populate your container registry, you need to push a container image to it.
+By default, private {HubName} does not include container images. In order to populate your container registry, you need to push a container image to it. You can follow the procedures in this section to learn to pull images from the Red Hat Ecosystem Catalog (registry.redhat.io), tag them, and push them to your private {HubName} container registry.
```
Comment (ariordan-redhat) [inline downstream/assemblies/hub/assembly-populate-container-registry.adoc:13]: Minimalism suggestions:

- Omit "In order"
- "The procedures in this section describe how to pull..."

File: downstream/assemblies/hub/assembly-populate-container-registry.adoc [hunk 2]
```diff
@@ -10,13 +10,14 @@ ifdef::context[:parent-context: {context}]
...
 
 == Prerequisites
 
 * You have permissions to create new containers and push containers to private {HubName}.
```
Comment (ariordan-redhat) [inline downstream/assemblies/hub/assembly-populate-container-registry.adoc:17]: Is private automation hub a proper noun? Or should it be "puh to your private {HubName}?

File: downstream/modules/automation-hub/proc-obtain-images.adoc [hunk 3]
```diff
@@ -0,0 +1,46 @@
+
+
+[id="obtain-images"]
+
+
+= Obtaining images for use in {HubName}
+
+
+[role="_abstract"]
+Before you can push container images to your private {HubName}, you must first pull them from an existing registry and tag them for use. This example details how to pull an image from the Red Hat Ecosystem Catalog (registry.redhat.io).
```
Comment (ariordan-redhat) [inline downstream/modules/automation-hub/proc-obtain-images.adoc:10]: This procedure ?

File: downstream/modules/automation-hub/proc-obtain-images.adoc [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
++
+-----
+$ podman login `registry.redhat.io`
+-----
++
+. Enter your username and password at the prompts.
+. Pull a container image:
++
+-----
+$ podman pull registry.redhat.io/[container image name]:[tag]
```
Comment (ariordan-redhat) [inline downstream/modules/automation-hub/proc-obtain-images.adoc:28]: Would angle brackets be correct here?

File: downstream/modules/automation-hub/proc-obtain-images.adoc [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+.Verification
+
+
+. List the images in local storage:
++
+-----
+$ podman images
+-----
++
+. Verify that the image you recently pulled is contained in the list.
```
Comment (ariordan-redhat) [inline downstream/modules/automation-hub/proc-obtain-images.adoc:41]: I looked at the podman docs - does the images command display the tags or does the command need a flag?

File: downstream/modules/automation-hub/proc-tag-pulled-image.adoc [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+[role="_abstract"]
+After you pull images from a registry, tag them for use in your private {HubName} container registry.
+
+.Prerequisites
+
+* You have pulled a container image from an external registry.
+
+.Procedure
+
+. Tag a local image with the {HubName} container repository
```
Comment (ariordan-redhat) [inline downstream/modules/automation-hub/proc-tag-pulled-image.adoc:18]: Bullet point for a single-step procedure

File: downstream/modules/automation-hub/proc-obtain-images.adoc (hunk 3 above)
Comment (ariordan-redhat) [inline downstream/modules/automation-hub/proc-obtain-images.adoc:10]: Do we put backticks around `registry.redhat.io` ?  OpenShift docs use them.

File: downstream/modules/automation-hub/proc-obtain-images.adoc [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+.Prerequisites
+
+* You have permissions to pull images from registry.redhat.io
+
+.Procedure
+
+. Log in to Podman using your registry.redhat.io credentials:
++
+-----
+$ podman login `registry.redhat.io`
```
Comment (ariordan-redhat) [inline downstream/modules/automation-hub/proc-obtain-images.adoc:21]: I don't think the backticks should be in the command
https://access.redhat.com/RegistryAuthentication

File: downstream/modules/automation-hub/proc-obtain-images.adoc (hunk 3 above)
Comment (ariordan-redhat) [reply downstream/modules/automation-hub/proc-obtain-images.adoc:10]: But the Container Registry Authentication docs don't add backticks to inline mentions of registry.redhat.io

File: downstream/assemblies/hub/assembly-populate-container-registry.adoc (hunk 2 above)
Comment (cbudz) [reply downstream/assemblies/hub/assembly-populate-container-registry.adoc:17]: component names are not treated as proper nouns, though it looks strange in the documentation.

File: downstream/modules/automation-hub/proc-obtain-images.adoc (hunk 3 above)
Comment (cbudz) [reply downstream/modules/automation-hub/proc-obtain-images.adoc:10]: Added

**Comment by cbudz** [comment]
@ariordan-redhat I've made updates here and rebuilt the preview so you can see them. One attribute is presently missing that I need to backport, but will do so in a separate PR

**Pull request merged**